website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -448,6 +450,294 @@ $ consul-k8s proxy read backend-658b679b45-d5xlb -o raw
+ext_authz                       info
+rbac                            info
+decompression                   info
+envoy_bug                       info
+file                            info
+pool                            info
+```
+
+Update the log levels for all loggers of an Envoy proxy in a given pod.
+```shell-session
+$ consul-k8s proxy log server-697458b9f8-4vr29 -u warning
+Envoy log configuration for server-697458b9f8-4vr29 in namespace default:
+
+==> Log Levels for server-697458b9f8-4vr29
+Name                            Level
+pool                            warning
+rbac                            warning
+tracing                         warning
+aws                             warning
+cache_filter                    warning
+decompression                   warning
+init                            warning
+assert                          warning
+client                          warning
+misc                            warning
+udp                             warning
+config                          warning
+hystrix                         warning
+key_value_store                 warning
+runtime                         warning
+admin                           warning
+dns                             warning
+jwt                             warning
+redis                           warning
+quic                            warning
+alternate_protocols_cache       warning
+conn_handler                    warning
+ext_proc                        warning
+http                            warning
+oauth2                          warning
+ext_authz                       warning
+http2                           warning
+kafka                           warning
+mongo                           warning
+router                          warning
+thrift                          warning
+grpc                            warning
+matcher                         warning
+hc                              warning
+multi_connection                warning
+wasm                            warning
+dubbo                           warning
+filter                          warning
+upstream                        warning
+backtrace                       warning
+connection                      warning
+io                              warning
+main                            warning
+happy_eyeballs                  warning
+rds                             warning
+tap                             warning
+envoy_bug                       warning
+rocketmq                        warning
+file                            warning
+forward_proxy                   warning
+stats                           warning
+health_checker                  warning
+lua                             warning
+secret                          warning
+quic_stream                     warning
+testing                         warning
+websocket                       warning
+```
+
+Update the log levels for specific loggers of an Envoy proxy in a given pod.
```

Comment:
```suggestion
The following command updates the `grpc` log level to `error`, the `http` log level to `critical`, and the `runtime` log level to `debug` for pod ID `server-697458b9f8-4vr29`.
```
We can probably leave off the bit about the pod ID because the important information in the example is demonstrating the syntax for updating specific loggers.